helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -147,6 +147,12 @@ def _onchange_partner_id(self):
             self.partner_name = self.partner_id.name
             self.partner_email = self.partner_id.email
 
+    def _get_default_email_channel(self):
```

Comment:
Why creating this extra method?

Reply:
I can't say for sure as I'm not the original author, but my guess would be that this method was made to be overridable for custom email channels

Reply:
But the way to do that is with default values in the alias, and the other method is already prepared for that, so better to remove it. Let's not bloat the module.

Reply:
I you deem it unnecessary I can remove it

Reply:
Yes

Reply:
It's not removed yet.

Reply:
Yeah I'm sorry, I had just removed its use but not the method itself